cfgov/unprocessed/css/organisms/video-player.less

```diff
@@ -83,39 +82,29 @@
     display: block;
     opacity: 0;
     transition: opacity .25s;
+    // Needed for IE11 to not stretch the image.
+    flex-shrink: 0;
 
-    .no-js & {
-        position: absolute;
-        top: 0;
-        right: 0;
-        width: 100%;
-        height: 100%;
-        background: #fff;
-        display: flex;
-
-        img {
-            padding: 10px;
-            box-sizing: border-box;
-            align-self: center;
-        }
+    // Using the default CFPB cover card image.
+    &[src="/static/img/cfpb_video_cover_card_954x200.png"] {
```

Comment:
This can't be exact because filenames have cachebusting random strings appended in production.

Reply:
Fixed in https://github.com/cfpb/consumerfinance.gov/pull/6284/commits/ed57f474b42f7c1cffa8b126931b185542baf46a